### DISCUSSION TITLE
Company specific rules

### DISCUSSION THREAD

**Discussion by wtrocki** [graphql-rules/graphql-rules#118] [Ideas]
Do you guys accept rules from different companies? 
I would agree with most of the suggestions but for some of the cases. 
I would really love to use this website as an entry point for anyone who wants to build a graphql schema. Do you think it is possible to extend it with company-specific rules.

**Comment by nodkz** [comment]
Hi @wtrocki 
Welcome on board! 👌

Of course, we accept new rules and suggestions for the improvement of current rules.
Our aim to create a community-driven platform for gathering experience and recommendations in schema construction.

For now, we are in deeeep beta. And most procedures (eg. how to add new rules) are not implemented.
So for now just open a new issue per every your rule. And we will discuss them there. Thanks 🙏

PS. In future, I want to introduce a mechanism which allows picking desired rules to your collection for sharing with your team as a convention.
